docs/marketplace/fastlane/enterprise-store/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Setting Up Appcircle Enterprise Store Plugin
+sidebar_label: Enterprise App Store
+description: Enhance powerful action to publish your builds to appcircle app store
+tags:
+  [
+    testing-distribution,
+    overview,
+    concepts,
+    app store,
+    internal testing,
+    beta testing,
+    binary distribution,
+    ipa distribution,
+    apk distribution,
+  ]
+sidebar_position: 2
+---
+
+The Appcircle Enterprise Store plugin allows users to publish their apps and start distribution to test groups or individuals.
+
+### Discover Action
+
+You can discover more about this action and install it by:
+https://rubygems.org/gems/fastlane-plugin-appcircle_enterprise_store
```

Comment:
You can learn more about this action and install it by visiting [here.](https://rubygems.org/gems/fastlane-plugin-appcircle_enterprise_store)